stock_reserve_sale/wizard/sale_stock_reserve_view.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_sale_stock_reserve_form" model="ir.ui.view">
+        <field name="name">sale.stock.reserve.form</field>
+        <field name="model">sale.stock.reserve</field>
+        <field name="arch" type="xml">
+            <form string="Reserve Stock">
+                <p class="oe_grey">
+                    A stock reservation will be created for the products
+                    of the selected quotation lines. If a validity date is specified,
+                    the reservation will be released once the date has passed.
+                </p>
+                <group>
+                    <field name="location_id" />
+                    <field name="location_dest_id" />
```

Comment:
I'm also thinking if we should enable the possibility to choose the location in which the product will be reserved, I don't see any case for which we shouldn't use the virtual location created in `stock_reserve`